lib/sublime-block-comment.js

```diff
@@ -11,15 +11,31 @@ export default {
     const buffer = editor.getBuffer();
 
     buffer.transact(() => {
-      for (const selection of editor.getSelectionsOrderedByBufferPosition().reverse()) {
-        const range = selection.getBufferRange();
-        const textInRange = buffer.getTextInRange(range);
-
+      for (let i in editor.getSelectionsOrderedByBufferPosition().reverse()) {
+        const selection = editor.getSelectionsOrderedByBufferPosition().reverse()[i];
```

Comment:
I don't quite understand the motivation behind this change:

```diff
-      for (const selection of editor.getSelectionsOrderedByBufferPosition().reverse()) {
+      for (let i in editor.getSelectionsOrderedByBufferPosition().reverse()) {
+        const selection = editor.getSelectionsOrderedByBufferPosition().reverse()[i];
```

They seem equivalent as far as I can see.